api.go

```diff
@@ -315,6 +316,11 @@ type (
 		CatalystPipelineStrategy string    `json:"catalystPipelineStrategy,omitempty"`
 	}
 
+	DeleteAssetTaskParams struct {
+		Id            string `json:"id"`
```

Comment:
~~Can you specify what is this id? I assume it's not the task id.~~

Actually, I don't think you need to specify. You can just remove all these fields. The input (and if relevant, output) assets are already specified in the root of the task object and should not be repeated in the params, as that'd be redundant and lead to code repetition (like not reusing the existing logic that fetches all the task context (check comment below)).